src/odo/odoWrapper.ts

```diff
@@ -194,13 +194,12 @@ export class Odo {
         );
     }
 
-    public async analyze(currentFolderPath: string): Promise<AnalyzeResponse[]> {
+    public async alizerAnalyze(currentFolderPath: Uri): Promise<AlizerAnalyzeResponse[]> {
```

Comment:
It might be a good idea to move this function to it's own file, since it no longer uses odo, and this file contains wrappers around odo commands